Documents/panel.md

```diff
@@ -1,17 +1,29 @@
-# Panel CSV Parser
+# Panel CSV Loader
 
 ## Overview
-The panel parser is designed to read a CSV file and load the data onto the HMDA-Platform.  The CSV file should use the `|` (pipe) delimiter, and should include a header row as the first line.
+The panel loader is designed to read a CSV file and load the data onto the HMDA-Platform.  The CSV file should use the `|` (pipe) delimiter, and should include a header row as the first line.
+
+## Environment Variables
+There are two environment variables used by the panel loader.  Both must be set correctly in order for the data to be sent to the admin API.
+
+For testing locally, no changes need to be made.  The defaults for both of these variables will point to the correct local admin API.
+
+For loading panel data into the cluster, navigate to a running `hmda-platform` service in DC/OS.  Under the "Endpoints" section you'll find the IP address and three ports.  As of now, the middle port will always be the admin API.  In your terminal, set both of the environment variables to the DC/OS values.
```

Comment:
The ports are declared in the `application.conf` file for the api project.  I don't know how likely this is to change, but as of now, the middle port (8081, when running locally) is the admin api.

When you navigate to a running service in DC/OS, you'll find a details page that lists configuration settings for that service.  One section is called "endpoints".